data/persist.js

```diff
@@ -30,6 +35,25 @@ export function withRehydratation( reducer, reducerKey, storageKey ) {
 	return enhancedReducer;
 }
 
+/**
+ * Export withRehydratation (a misspelling of withRehydration) for backwards
+ * compatibility.
+ *
+ * @param {Function} reducer    The reducer to enhance.
+ * @param {string}   reducerKey The reducer key to persist.
+ * @param {string}   storageKey The storage key to use.
+ *
+ * @return {Function} Enhanced reducer.
+ */
+export function withRehydratation( reducer, reducerKey, storageKey ) {
+	deprecated( 'wp.data.withRehydratation', {
+		version: '3.1',
```

Comment:
It should be rather 3.2. It should be also included in the `Deprecated` document: https://github.com/WordPress/gutenberg/blob/master/docs/reference/deprecated.md.